test/system/800-config.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Test specific configuration options and overrides
+#
+
+load helpers
+
+@test "podman CONTAINERS_CONF - CONTAINERS_CONF in conmon {
+    skip_if_remote "can't check conmon environment over remote"
+
+    # Make an innocuous containers.conf in a non-standard location
+    export CONTAINERS_CONF="$PODMAN_TMPDIR/containers.conf"
```

Comment:
Please don't do this. Instead, do:
```
    conf_tmp="$PODMAN_TMPDIR/containers.conf"
    cat >$conf_tmp ...

   ...

   CONTAINERS_CONF=$conf_tmp run_podman .....
```
Reason: `export`ing will not propagate across tests, but it _will_ carry over into `teardown()`. If for any reason the resulting conf file is bad, the `podman` invocations in `teardown` will fail, and cascading test failures will ensue.

Reply:
Updated as suggested.